pkg/nanopb/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME=nanopb
+PKG_URL=https://github.com/nanopb/nanopb.git
+PKG_VERSION=be0dd4b3aacdc9c2bbe21568b6a37218f2bb959a
+PKG_LICENSE=zlib
+
+.PHONY: all
+
+RIOT_CFLAGS = $(INCLUDES) -std=c99
+
+TOOLCHAIN_FILE = $(PKG_BUILDDIR)/xcompile-toolchain.cmake
+
+PKG_DIR=$(CURDIR)
+PROTO_DIR=$(APPDIR)/proto
+PROTO_BUILD_DIR=$(APPDIR)/proto_compiled
+PROTO_FILES=$(shell ls $(PROTO_DIR))
+
+all: $(PKG_BUILDDIR)/Makefile
+	cp $(PKG_BUILDDIR)/libnanopb.a $(BINDIR)/nanopb.a
+
+$(PKG_BUILDDIR)/Makefile: $(TOOLCHAIN_FILE)
```

Comment:
Because this depends on  $(TOOLCHAIN_FILE), which depends on git-download which is a PHONY target, this rules is being executed every time.